### DISCUSSION TITLE
Registering new app

### DISCUSSION THREAD

**Discussion by stackiflowi** [etsy/open-api#1060] [Q&A]
Hi,

I'm trying to register with a new Etsy account for the API keys with my company. On this page:

https://www.etsy.com/developers/register

I fill out everything which is purple / pink. The application is a Seller Tool and only for myself or colleagues, not commercial and only for Upload / edit Listings.

Immediately I got a keystring and a shared secret with the notice that:

"NOTE: YOUR KEY IS NOT YET ACTIVE
We will be reviewing your key shortly. If it is approved it will be updated here. Notifications are not provided for requests that are not approved."

And that's is the problem. For 2 weeks nothing happens and after that this page gots deleted and I can start it again. How should I fill the datas to get accepted? I don't know much about what's going on behind the scenes.

Could anyone give me some information how to deal with this?

Thanks.

**Comment by stackiflowi** [comment]
Anyone would have a hint on this?

It's kinda annoying that I can't get any information, and I can't move on.



Thanks.

**Reply by brettcthompson25** [reply]
having the same issue....

**Reply by Archieversam** [reply]
Are you looking for a way to track and convert majority of your traffic into potential buyers in an easy way? Kindly contact us now for more details or use the link below. THANKS.
https://cutt.ly/Sw0vSP70

**Comment by teophilus** [comment]
Haveing the same issue, I keep getting rejected

**Reply by Archieversam** [reply]
Are you looking for a way to track and convert majority of your traffic into potential buyers in an easy way? Kindly contact us now for more details or use the link below. THANKS.
https://cutt.ly/Sw0vSP70

**Comment by muaz742** [comment]
Hi,

I was rejected three times.

While searching the internet, I came across a comment that initiated the application process with "personal access" and then transitioned it into commercial use.
I'm trying the same approach.

My personal access application has now been approved. I am developing my software with my "personal access" authorization. I haven't yet applied for "commercial access," but the application link is active.

I recommend that you first apply for personal access and then proceed to apply for commercial use using this application.

Happy coding.

**Reply by brettcthompson25** [reply]
Is there a different application for personal and commercial?

**Reply by simonj** [reply]
@brettcthompson25 Yes personal your request has a limit and you can not charge money for your users

**Reply by Archieversam** [reply]
Are you looking for a way to track and convert majority of your traffic into potential buyers in an easy way? Kindly contact us now for more details or use the link below. THANKS.
https://cutt.ly/Sw0vSP70

**Comment by simone2424** [comment]
Start with personal access then wait a week, apply for commercial but from the start do not state that you are using 3rd party access! This will get a rejection status!

**Reply by Archieversam** [reply]
Are you looking for a way to track and convert majority of your traffic into potential buyers in an easy way? Kindly contact us now for more details or use the link below. THANKS.
https://cutt.ly/Sw0vSP70

**Comment by mifloma** [comment]
@simone2424 Did you use your seller account for the request or did you create a developer account? The idea with personal access first only works using a seller account, right?

I have the same problem. I want to integrate the api with my custom Etsy Seller tool. My requests have been rejected three or four times. Reason: "other". Unfortunately also personal access for my seller acoount was rejected yesterday 😑

**Reply by constantin-bogdan** [reply]
@mifloma 
I also got rejected. They told me that I am not allowed to use the word "etsy" in my app name. I am also waiting for 20 days now to get aprooval to my second try.

**Reply by mifloma** [reply]
It really takes time because every try takes 20+ days for approval/rejection. The good thing is, avoiding the word "etsy" in your app name seems to be an achievable task 😄 Good luck for your second attempt. How did you find out, that they don't like your app name? Did you contact the developer support via email?

**Reply by constantin-bogdan** [reply]
I got rejected again, now they say my app "Circumvents Etsy checkout process" although I just want to import my Etsy Sales into my ERP System to get them into production. They are making it really really hard to use Etsy API, so dissapointing.

**Reply by mifloma** [reply]
That is really annoying. Maybe you need to rethink your idea a bit so as not to violate the Etsy API guidelines. Do you already have an idea of what to do next?

My problem is still that Etsy doesn't reply to my emails. I still don't know why my requests are being rejected. But it could be similar to yours.

**Comment by constantin-bogdan** [comment]
Yes, I contacted dev support and they replied in 5-6 days. We will see if
it's enaugh to avoid "etsy" in the app's name.

**Reply by verriterri** [reply]
Did you get approved? I'm about to request access as well and want to avoid as much back and forth as possible.

**Comment by cyberjo007** [comment]
etsy support complete shit

**Comment by bringphototolife** [comment]
how is it going?
I got the same trouble
